### PR TITLE
Fix wrong refs to event type

### PR DESCRIPTION
**Pull request by aliok**
Fix forgotten updates to event type

### CODE REVIEW

File: pkg/apis/samples/v1alpha1/samplesource_types.go [hunk 1]
```diff
@@ -61,11 +61,6 @@ var _ kmeta.OwnerRefable = (*SampleSource)(nil)
 // Check that SampleSource implements the Conditions duck type.
 var _ = duck.VerifyType(&SampleSource{}, &duckv1.Conditions{})
 
-const (
```
Comment (aliok) [inline pkg/apis/samples/v1alpha1/samplesource_types.go:64]: This is not used and it is not updated

File: config/300-samplesource.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ metadata:
   annotations:
     registry.knative.dev/eventTypes: |
       [
-        { "type": "dev.knative.sample.source" }
+        { "type": "dev.knative.sample" }
```
Comment (aliok) [inline config/300-samplesource.yaml:25]: https://github.com/knative/sample-source/blob/master/pkg/adapter/adapter.go#L55

**Comment by chaodaiG** [comment]
Kick tide
/test all

**Pull request merged**